deployments/fpga_plugin/intel-fpga-initcontainer.Dockerfile

```diff
@@ -0,0 +1,79 @@
+### 1. Temporary image to prepare AOCL and OPAE components
+FROM centos:centos7.4.1708 as temporary
+
+# install aocl and opae deps
+RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+RUN yum -y install which perl uuid libuuid-devel json-c
+
+ADD a10_gx_pac_ias_1_1_pv_rte_installer.tar.gz $WORKDIR
```

Comment:
Is `$WORKDIR` set implicitly?

Reply:
I think so. Whatever it is I'm ok with it as this is a temporary image.